assets/js/googlesitekit/modules/datastore/__fixtures__/list.json

```diff
@@ -22,68 +22,46 @@
 		"homepage": "https://search.google.com/search-console",
 		"internal": false,
 		"order": 1,
+		"forceActive": true,
+		"recoverable": false,
+		"shareable": true,
 		"active": true,
 		"connected": true,
 		"dependencies": [],
 		"dependants": [],
-		"owner": null,
-		"forceActive": true,
-		"shareable": true,
-		"recoverable": false
+		"owner": null
 	},
 	{
-		"slug": "adsense",
-		"name": "AdSense",
-		"description": "Earn money by placing ads on your website. It’s free and easy.",
-		"homepage": "https://adsense.google.com/start?source=site-kit&url=http://example.com",
+		"slug": "ads",
+		"name": "Ads",
+		"description": "Track conversions for your existing Google Ads campaigns",
+		"homepage": "https://google.com/ads",
 		"internal": false,
-		"order": 2,
+		"order": 1,
+		"forceActive": false,
+		"recoverable": false,
+		"shareable": false,
 		"active": false,
 		"connected": false,
 		"dependencies": [],
 		"dependants": [],
-		"owner": null,
-		"forceActive": false,
-		"shareable": true,
-		"recoverable": false,
-		"features": [
-			"Monetize your website",
-			"Intelligent, automatic ad placement"
-		]
```

Comment:
`features` are provided during client-side registration, although this list should only include module definition data from the server side.